src/maker/config.rs

```diff
@@ -193,6 +193,50 @@ impl MakerConfig {
             .unwrap_or(default_config.connection_type),
         })
     }
+
+    // Method to manually serialize the Maker Config into a TOML string
+    pub fn to_toml_string(&self) -> String {
+        format!(
+            r#"
+        port = {}
```

Comment:
Just the Toml string isn't very useful as for all cases we immediately gonna write it in a file. So lets just put that fs::write line also in the API. Will simplify the calling sites.

Reply:
Unless you can think of some use case where we will get the toml string but not write the file.